### DISCUSSION TITLE
how to change the path for model in subtitle edit

### DISCUSSION THREAD

**Discussion by amirbahalegharn** [Purfview/whisper-standalone-win#7] [Q&A] [answered]
first of all,thanks
but how to specify the path to the model downloaded? after renaming the file to ctanslator, i select ctanslator2 in subedit and browse to your file, but in choose model , it's not showing the big model i downloaded whether i put it in "_model" or "models" or" model" or "_models" anywhere in subedit,whisper folder there or in whisperfaster,or in ctanslator folder inside it....none worked and the open folder model option in subedit points to .cache\whisper-ctranslate2" within the "user data" directory which i want to change it elsewhere not drive C. would you enlighten me where and with  how-to-do ? also with faster performance, has those repeated hallucinations without transcribing anything been improved as well or not?
thanks in advance

**Comment by Purfview** [comment, marked as answer]
But this you should ask in Subtitle Edit repo. :)

At the moment there is no option to select models folder in SE. So you need to download them with SE to be able to select them.
In some SE beta you could select any model without downloading them...

Btw, with Faster-Whisper you can expect better results if you don't use it in SE. [because of SE's audio processing]
